src/linear_algebra/determinant.lean

```diff
@@ -319,4 +330,80 @@ begin
     exact hkx }
 end
 
+/-- The determinant of 2x2 block matrix with the lower left block zero, is the product of
+the determinants of the diagonal blocks. For the generalization to any number of blocks
+see upper_block_triangular_det in linear_algebra/matrix.lean  -/
+lemma upper_two_block_triangular_det (A  : matrix m m R) (B  : matrix m n R) (D  : matrix n n R) :
```

Comment:
```suggestion
/-- The determinant of a 2x2 block matrix with the lower-left block equal to zero is the product of
the determinants of the diagonal blocks. For the generalization to any number of blocks, see
`matrix.upper_block_triangular_det` in `linear_algebra/matrix.lean`. -/
lemma upper_two_block_triangular_det (A : matrix m m R) (B : matrix m n R) (D : matrix n n R) :
```

Reply:
I'd probably omit the filename, viewers of the web docs can click the definition name, and local users can use `#check` to find it.

Reply:
Yeah, I agree:
```suggestion
/-- The determinant of a 2x2 block matrix with the lower-left block equal to zero is the product of
the determinants of the diagonal blocks. For the generalization to any number of blocks, see
`matrix.upper_block_triangular_det`. -/
lemma upper_two_block_triangular_det (A : matrix m m R) (B : matrix m n R) (D : matrix n n R) :
```